.github/workflows/ci.yml

```diff
@@ -91,7 +91,7 @@ jobs:
         - name: setup-python
           uses: actions/setup-python@v2
           with:
-            python-version: 3.6
+            python-version: 3.10.8
```

Comment:
If you set it to 3.10 it will automatically select the latest subversion

Reply:
That is convenient!